app/services/pledge_form_generator.rb

```diff
@@ -92,6 +92,18 @@ def generate_pledge_pdf
       contact_email: 'info@yellowhammerfund.org',
       billing_email: 'billing@yellowhammer.org',
       img: 'yhf_logo.png'
+    },
+    'Test Fund' => {
```

Comment:
Is it necessary to check this in? This doesn't actually change the pledge itself, right?

Reply:
ah, prob not. the issue is that without a `pledge_generation_config` none of the pledge stuff shows up, so i couldn't debug. I can

1. remove it entirely?
2. or set catfund to use one of DCAF or YHF.

Reply:
oh! I see what you mean here.

I would bet that this has utility outside of funds having pledge_generation_config. What do you think about moving the new stuff in the pledge partial it to step 3 above the 'I sent the pledge' checkbox and reverting this change? Seems like 'having the number to dial on hand' lends itself well to the sending step. Thoughts?

Reply:
ah, sure. let me have a look - i think i know what you mean

Reply:
ok, moved the contact info down, so it will show up regardless.